### ISSUE TITLE
Stanley Knife or Scalpel

### ISSUE THREAD

**Issue by adrientetar** [Templarian/MaterialDesign#864]
I develop an open-source typeface design software which uses icons in this set (thanks!). There are quite some vector app icons in there, but a stanley knife or scalpel icon is sorely missing for cut curve tools.

**Comment by CoDEmanX** [comment]
Stanley knife? We call that carpet cutter in German :yum: 

It's a difficult one... maybe better go for the scalpel.
![cutter](https://cloud.githubusercontent.com/assets/1309249/11405043/ad301218-93a5-11e5-9aac-50a6da7acf49.png)
![cutter2](https://cloud.githubusercontent.com/assets/1309249/11405042/ad2e7cc8-93a5-11e5-85d4-6c70da45945b.png)

**Comment by adrientetar** [comment]
@CoDEmanX The first one looks pretty good, is it available somewhere? I use SVGs preferrably.  
The second one might be passing as well hm, I'd have to try in the app.

Also it just occured to me that nib icon would be useful as well, opened #867 for that.

By the way, we just say "cutter" in French heh, no need to get fussy!

**Comment by Templarian** [comment]
I like the icon, but not for a slice tool. Exacto knife, would be better.

![image](https://cloud.githubusercontent.com/assets/338885/11406314/46edfb1a-9372-11e5-9f9c-56cfa77be69b.png)

**Comment by adrientetar** [comment]
So that's how this is called... thanks. Scalpel/stanley knife icon would already be miles bette rthan what I currently use (scissors).

**Comment by CoDEmanX** [comment]
Exacto knife :smile: Another awesome word! We should really have that as icon, probably requested here already: #665 

@adrientetar: I added download links to both icons above.

What I don't like about the first icon is that the blade doesn't look very sharp. It needs some fine tuning to be better recognizable at larger scales as well (make the handle more organically shaped?)

**Comment by adrientetar** [comment]
@CoDEmanX Thanks, much appreciated. Here's how it looks right now:

![foo](https://cloud.githubusercontent.com/assets/1250894/11409165/45c34960-93be-11e5-85fd-292a9e70fc88.PNG)

**Comment by adrientetar** [comment]
Yeah the blade of the first one would maybe need fine-tuning of some sort. Anyway, thanks again. At least now I won't sigh everytime I open my app.

**Comment by Templarian** [comment]
So we have `box-cutter`. Which needs an alias added for `stanely-knife`. Not sure the design for the exacto knife works. Needs to look more like the metal `scalpel`.

**Comment by PeterShaggyNoble** [comment]
Alias `stanley-knife` added to `box-cutter`.

**Comment by goyney** [comment]
It looks like `box-cutter` was already added. The rest of this issue is hard to follow. If there is another type of knife icon you need, please open a new issue for it. Thanks! 😄
